lms/static/certificates/sass/_print.scss

```diff
@@ -0,0 +1,326 @@
+// ------------------------------
+// // Open edX Certificates: Print Specific Styling
+
+// About: styling specific for print layouts.
+
+// #UTILITIES
+// #BASE
+// #LAYOUT
+// #ELEMENTS
+// #CASES
+
+// ------------------------------
+// #UTILITIES
+// ------------------------------
+
+// page set up - only available to modern browsers now
+@page {
+    size : landscape;
```

Comment:
NOTE: defines how the page should be printed, our certs are designed for landscape. Sadly, this only works in webkit browsers now. :/